decidim-core/spec/system/user_profile_spec.rb

```diff
@@ -122,6 +122,40 @@
           expect(page).not_to have_content(translated(non_public_resource.title))
         end
       end
+
+      context "when the user follows a blocked user" do
+        let(:blocked_user) { create(:user, blocked: true) }
```

Comment:
```suggestion
        let(:blocked_user) { create(:user, :blocked) }
```

Is it possible to use trait `:blocked` as the other test.